src/main/java/duke/ToDo.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+/**
+ * Inherits from the Task class. Represents a to do task.
+ * ToDo object stores task description and done status. A ToDo object is marked as not done by default.
+ * A ToDo object is able to be set as done by calling setDone() method from the parent class.
+ */
+public class ToDo extends Task {
+    /**
+     * A ToDo object uses its parent class constructor
+     * @param description description of to do task
```

Comment:
Perhaps punctuation behind each parameter description could make it look neater

Reply:
Resolved this. Thank you!